mlflow/store/model_registry/dbmodels/models.py

```diff
@@ -170,3 +170,27 @@ def __repr__(self):
     # entity mappers
     def to_mlflow_entity(self):
         return ModelVersionTag(self.key, self.value)
+
+
+class SqlRegisteredModelAlias(Base):
+    __tablename__ = "registered_model_aliases"
+    name = Column(String(256), nullable=False)
+    alias = Column(String(256), nullable=False)
```

Comment:
This is 250 in other places.  Why 256 here?

Reply:
Good catch I think we updated this value but forgot to change the rest, let me do that now